packages/product-exporter/src/main.js

```diff
@@ -95,6 +95,9 @@ export default class ProductExporter {
           this.logger.debug(
             `${products.length} products written to outputStream`
           )
+          if (products.length < 1) {
+            return Promise.reject(Error('No products found'))
```

Comment:
You should reject with a `new` error instance here

Reply:
There's actually no need for that as they both do the same
> When Error is used like a function -- without new, it will return an Error object. Therefore, a mere call to Error will produce the same output that constructing an Error object via the new keyword would.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error